README.rst

```diff
@@ -101,6 +101,16 @@ can automatically run the tests for various python versions like this:
 
     tox
 
+Update pyptest version
+++++++++++++++++++++++
+
+In the following files the pytest version needs to be changed:
```

Comment:
This sentence could be hard to understand without context. How about something along "If you update or change the pytest version, the following files need to be changed"?

Reply:
Done